In [21]:
import pandas as pd

data = pd.read_csv(
    'SUP_ALUNO_2019.csv',
    encoding = "ISO-8859-1", 
    usecols=['CO_IES', 'TP_COR_RACA'],
    sep='|'
    )

data.head()


CO_IES  TP_COR_RACA
0       1            2
1       1            3
2       1            3
3       1            2
4       1            3

In [22]:
num_of_stds = data.loc[data['CO_IES'] == 587].count()[0]
print(num_of_stds)

18643


## Utilizando o código IES da UFRPE verificamos que 18643 alunos foram registrados

In [44]:
stds = data.loc[data['CO_IES'] == 587]

B = stds.loc[stds['TP_COR_RACA'] == 1].count()[0]
P = stds.loc[stds['TP_COR_RACA'] == 2].count()[0]
Pa = stds.loc[stds['TP_COR_RACA'] == 3].count()[0]
A = stds.loc[stds['TP_COR_RACA'] == 4].count()[0]
In = stds.loc[stds['TP_COR_RACA'] == 5].count()[0]

no_info = num_of_stds - (B+ P+ Pa+ A +In)



861
